local-tests/test_recover.py

```diff
@@ -0,0 +1,64 @@
+#!/bin/env python
```

Comment:
No idea how to set finality version to `CURRENT` in the test, so left a comment below that this test should be ran only after switching the finality version. Tho I ran it manually setting the finality version.
* fails with `LEGACY` (as expected)
* works with `CURRENT` (as expected)

Reply:
Is this test related to changing abft versins? If not pls add it in another pr

Reply:
sidenote: use scheduleFinalityVersion extrinsic from pallet Aleph to change the finality version.

Reply:
The test verifies that the node with new abft is resilient in case of a particular scenario.
How can we use `scheduleFinalityVersion` in a python local test?

Reply:
I changed the test so now it:
* Bumps the abft version using extrinsic at some session
* runs the nodes for twice as many sessions
* performs the recover scenario afterwards (kills all nodes, recovers all but one, and expects that the living nodes finalize)

I noticed [here](https://github.com/Cardinal-Cryptography/aleph-node/blob/main/pallets/aleph/src/lib.rs#L310) that we always start the chain with legacy abft version, so updating abft using extrinsic will always be valid (we'll always go from legacy to current). This way this test will work now and even if we update abft further in the future. For this version of the code, calling this extrinsic is required to pass the test.